pkg/reporegistry/reporegistry.go

```diff
@@ -118,6 +120,12 @@ func (r *RepoRegistry) DistroHasRepos(distro, arch string) ([]rpmmd.RepoConfig,
 	return repos, nil
 }
 
+// (deprecated) DistroHasRepos is just here for compatbility with weldr which could
+// use `ReposByArchName(distro, arch, true)` instead.
+func (r *RepoRegistry) DistroHasRepos(distro, arch string) ([]rpmmd.RepoConfig, error) {
+	return r.reposByDistroArch(distro, arch)
+}
+
```

Comment:
So why keep it and not fix the Weldr API to use `ReposByArchName()`?

Reply:
Excellent point, I should have linked tohttps://github.com/osbuild/osbuild-composer/pull/4910 - sorry for that. Once that is in I can drop this compat wrapper.